clang/test/SemaHLSL/standard_conversion_sequences.hlsl

```diff
@@ -4,9 +4,8 @@
 void test() {
   
   // CHECK: VarDecl {{.*}} used f3 'vector<float, 3>':'float __attribute__((ext_vector_type(3)))' cinit
-  // CHECK-NEXt: ImplicitCastExpr {{.*}} 'vector<float, 3>':'float __attribute__((ext_vector_type(3)))' <VectorSplat>
-  // CHECK-NEXt: ImplicitCastExpr {{.*}} 'float' <FloatingCast>
-  // CHECK-NEXt: FloatingLiteral {{.*}} 'double' 1.000000e+00
+  // CHECK-NEXT: ImplicitCastExpr {{.*}} 'vector<float, 3>':'float __attribute__((ext_vector_type(3)))' <VectorSplat>
```

Comment:
It looks like we lost a CHECK line here. Is that expected?

Reply:
https://github.com/llvm/llvm-project/pull/93193/files#diff-49956cafb2ae707ec035858ec16fe0c8742aef703f29ce83339abf206c25080e in original diff there 3 of checks and test (looks like) passes

Reply:
Yes this is intentional, as the test doesn't pass without the change. Since #91015 the literal is a `float`, not a `double`, so the implicit cast to float doesn't happen.

Reply:
The log is truncated so we can't see the actual error, but you can see that this test is listed in the failures in the build for #93193 here: https://buildkite.com/llvm-project/github-pull-requests/builds/69357#018fdf40-9615-457d-93d7-b23bc0726a57/6-20940